stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/Messages.kt

```diff
@@ -105,16 +84,7 @@ public fun Messages(
     onScrolledToBottom: () -> Unit,
     modifier: Modifier = Modifier,
     contentPadding: PaddingValues = PaddingValues(vertical = 16.dp),
-    helperContent: @Composable BoxScope.() -> Unit = {
-        val coroutineScope = rememberCoroutineScope()
-        DefaultMessagesHelperContent(messagesState, lazyListState) {
-            coroutineScope.launch {
-                coroutineScope.launch {
```

Comment:
Was launching a coroutine in a coroutine here necessary or was it just a small mistake?

Reply:
mistake, no reason to launch a coroutine inside a coroutine